apps/site/src/views/Resources/components/ResourceCard/ResourceCard.module.scss

```diff
@@ -64,4 +60,49 @@
 .text {
 	width: 90%;
 	text-align: center;
+	font-family: utils.$font-family-base;
+}
+
+.islandBackground {
+	position: absolute;
+}
+
+.CardContainer {
```

Comment:
Nitpick: it might be good to have the cursor change to a pointer so users know the card is clickable.

Reply:
title is now clickable but the whole card may be a bit much (I think)